documentation/book/oauth/proc-oauth-client-config.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// assembly-oauth-strimzi-config.adoc
+
+[id='proc-oauth-client-config-{context}']
+= Configuring Kafka clients to use {oauth}
+
+This procedure describes how to configure Java-based Kafka clients to use {oauth} for interaction with Kafka brokers.
+
+Add a client callback plugin to your _pom.xml_ file, and configure the system properties.
+
+.Prerequisites
+
+* AMQ Streams and Kafka is running
+* {oauth} authorization server is deployed and configured for OAuth access to Kafka brokers
+* Kafka brokers are configured for {oauth}
+
+.Procedure
+
+. Add the client callback plugin to the `pom.xml` file for the Kafka client:
++
+[source,xml]
+----
+<dependency>
+ <groupId>io.strimzi</groupId>
+ <artifactId>kafka-oauth-client</artifactId>
+ <version>0.1.0.redhat-00001</version>
```

Comment:
We need to use the upstream version here.

```suggestion
 <version>0.1.0</version>
```

Reply:
Sorry. Missed that. I've set a variable for upstream and downstream.